source/docs/articles/sites/migrate.md

```diff
@@ -67,6 +67,7 @@ Manually import the site outside of our Importer Tool if any of the following ap
 - Your site exceeds file size limit for uploads.
 - Your site requires an upstream to an organizational or public distribution.
 - You would like to preserve the site's existing git history.
+- Your site is running Drupal 8 and was not created on Pantheon.
```

Comment:
@bmackinney importer was not tested, but probably should be. I went off the issue description in #995 for this update.

Do you have a test site or archive we can use?